src/components/text/text.module.css

```diff
@@ -24,8 +27,8 @@
 .bodyNormal,
 .bodyBig,
 .list {
-  color: var(--primary-black);
-  font-family: var(--font-britti-sans);
+  /*color: var(--primary-black);
+  font-family: var(--font-britti-sans);*/
```

Comment:
We can remove the color, but we should keep the font. Should not be commented out.